src/style.css

```diff
@@ -73,59 +78,81 @@ canvas:focus {
     -moz-osx-font-smoothing: grayscale;
 }
 
-/* these class names are not being used, are but included here to 
-keep track of what font glyphs are available.
-To use one, the span must have the "icon-moon" class name, and then 
-use the :after pseduo selector to input the content.
-*/
-.icon-focus:before {
-    content: "\e90a";
-    color: #d8d8d8;
+.ant-tooltip-inner,
+.ant-popover-inner {
+    background: var(--dark-theme-tooltip-bg) 0% 0% no-repeat padding-box;
+    box-shadow: 0px 3px 4px var(--black);
+    opacity: 1;
 }
 
-.icon-rotate:before {
-    content: "\e909";
+
+/* 
+Icomoon font glyphs are used for our custom icons, 
+they can be styled like text: font-size, color, etc.
+
+Most icons are used only once, or styled the same way in each use,
+so it is safe to apply style rules here, and then override in the
+rare case they need to be different.
+
+If new glyphs are added/glyphs are changed: update the css vars 
+in /styles/icons.css, and define the appropriate::after selector 
+here, apply icon using "name-icon" class name in conjunction 
+with "icon-moon" class name.
+ */
+
+ /* Current in-use icons */
+.star-empty-icon::after {
+    content: var(--star-empty);
+    color: var(--warm-gray)
```

Comment:
Existing rules defining global style rules were all unused with a comment that they were just for storage. Icons in use had their content  provided via code string  in stylesheets (i.e. content: "\e909") which gave no indication what the content was. This pattern defined here makes a lot of sense! I think we should use it all the time, not just for storing unused icons.